app/javascript/selectors/questionsSelector.js

```diff
@@ -158,3 +158,37 @@ function getAllActiveQuestionsFromRoots(rootQuestions, buildingId, state) {
 
   return questions;
 }
+
+export function getAllActiveQuestionIdsForCategory(categoryId, buildingId, state) {
```

Comment:
Hm I don't see the use for these two new selector methods -- isn't it the same code as `getAllActiveQuestions___` except it returns ids instead of the full objects?

If you want something with the same behavior as the old ones but only care about ids, you can do
```
getAllActiveQuestionsForCategory(...).map(question => question.id)
```